book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rent)
+    @rentals << rent
+  end
```

Comment:
- [ ] Good job on setting up the associations :+1:. I would recommend adding Getters and Setters for `@rentals`. Also, in the `add_rental` method you have to create the new rental achieving the `has-many` association. [ Hint: `Rental.new(date, self, person)`] (where the `self` is book itself that should be sent).

Reply:
Thank you @Dipeshtwis  👋 

I have implemented the required changes.

I will proceed to request another review now.

Cheers 🎉